### PR TITLE
Feature/5191 installable component

### PR DESCRIPTION
**Pull request by jesuspg**
@flopezag

### CODE REVIEW

File: .bumpversion.cfg [hunk 1]
```diff
@@ -11,3 +11,6 @@ replace = unreleased_version_label = "v{new_version}"
 search = sonar.projectVersion={current_version}
 replace = sonar.projectVersion={new_version}
 
+[bumpversion:file:setup.cfg]
+search = version = {current_version}
+replace = version = {new_version}
```
Comment (flopezag) [inline .bumpversion.cfg]: new line

File: fiwareskuld/impersonate.py [hunk 2]
```diff
@@ -135,7 +135,8 @@ def delete_trust_admin(self, trust_id):
         """Delete and invalidate a TRUST_ID
 
         WARNING: This method currently does not work.  It requires changes in
```
Comment (flopezag) [inline fiwareskuld/impersonate.py]: There is 2 blank spaces between "work." and "It requires..."

File: .travis.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ before_script:
 
 # command to run tests
 script:
-  nosetests --with-coverage --cover-package=./ --exe
+  cd tests/units;nosetests --with-coverage --cover-package=./ --exe
```
Comment (flopezag) [inline .travis.yml]: mmm, if you do it, are you sure that you are covering the fiwareskuld content?

Comment (jesuspg) [inline .travis.yml]: yes, I'm sure

**Comment by flopezag** [comment]
LGTM

**Pull request merged**